api/src/main/resources/jira.yaml

```diff
@@ -0,0 +1,137 @@
+  AdminSecurity:
+    type: basic
+
+##########################################################################################
+## PATHS
+##########################################################################################
+
+paths:
+ /issue:
+    post:
+      summary: Create issue in jira.
+      description: "Creates issue in JIRA and return IssueResponse"
+      tags:
+        - Jira
+      operationId: createIssue
+      parameters:
+        - name: fields
+          in: body
+          description: Attributes for creating issue
+          schema:
+            $ref: '#/definitions/IssueRequest'
+      responses:
+        200:
+          description: File saved
+          schema:
+            $ref: '#/definitions/IssueResponse'
+        400:
+          description: Bad request, check the summary/description field
+        503:
+          description: Cannot connect to JIRA. Please try again later
+      security:
+        - AdminSecurity: []
+
+ /issue/{issueKey}/attachments:
+   post:
+     summary: Add attachment to the jira ticket
+     tags:
+       - Jira
+     operationId: addAttachments
+     consumes:
+       - multipart/form-data
+     parameters:
+       - in: path
+         name: issueKey
+         type: string
+         required: true
+       - name: file
+         in: formData
+         description: Upload file as attachment
+         required: true
+         type: file
+       - in: header
+         name: X-Atlassian-Token
+         type: string
+         default: nocheck
+     responses:
+       200:
+         description: File saved
+       503:
+         description: Cannot connect to JIRA. Please try again later
+     security:
+       - AdminSecurity: []
+
+##########################################################################################
+## DEFINITIONS
+##########################################################################################
+definitions:
+  IssueRequest:
+      description: Details required for creating issue project info, summary, description etc
+      required:
+        - fields
+      properties:
+        fields:
+          $ref: '#/definitions/FieldsDetails'
+
+  FieldsDetails:
+      description: Fields information like project id, issue type etc
+      required:
+        - project
+      properties:
+        project:
+          $ref: '#/definitions/ProjectDetails'
+        summary:
+          type: string
+        description:
+          type: string
+        issuetype:
+          $ref: '#/definitions/IssueType'
+
+  ProjectDetails:
+      description: Project information
+      required:
+        - key
+      properties:
+        key:
+          type: string
+
+  IssueType:
+      required:
+        - name
+      properties:
+        name:
+          type: string
+
+  IssueResponse:
+      required:
+        - id
+        - key
+      properties:
+        id:
+          type: string
+        key:
+          type: string
+        self:
+          type: string
+
```

Comment:
The docs also mention Error responses: https://developer.atlassian.com/cloud/jira/platform/rest/#error-responses
Would it make sense to document that here and add a schema ref to it for the error cases above?